src/contract/contractFactory.ts

```diff
@@ -55,8 +61,8 @@ export class ContractFactory {
    *
    * @param providerOrAccount - new Provider or Account to attach to
    */
-  connect(providerOrAccount: ProviderInterface | AccountInterface): ContractFactory {
-    this.providerOrAccount = providerOrAccount;
+  connect(providerOrAccount: AccountInterface): ContractFactory {
```

Comment:
```suggestion
  connect(account: AccountInterface): ContractFactory {
```
should this also be renamed to account? Also in the comments above the `connect`

Reply:
466a2aacdac6034b6d5294c1e4ce1208b872acd8